src/utils/parseBalancer.ts

```diff
@@ -65,7 +67,16 @@ export function prepareBackendFromBalancer(rawBalancer: string) {
 
     // Use meta_backend if it is defined to form backend url
     if (window.meta_backend) {
-        return normalizePathSlashes(`${window.meta_backend}/${preparedBalancer}`);
+        const metaBackend = window.meta_backend;
+        const envPrefix = environment ? `/${environment}` : '';
+
+        // If meta_backend is a full URL (has protocol), don't add leading slash
```

Comment:
Please, update comment, it should point on the lack of env, not the leading slash